packages/kit/src/exports/vite/index.js

```diff
@@ -747,11 +742,14 @@ Tips:
 						}
 					},
 					publicDir: kit.files.assets
-					// TODO: enabling `experimental.enableNativePlugin` causes styles to not be applied
-					// see https://github.com/vitejs/rolldown-vite/issues/213
-					// experimental: {
-					// 	enableNativePlugin: true
-					// }
+				};
+			}
+
+			// @ts-expect-error `vite.rolldownVersion` only exists in `rolldown-vite`
+			if (vite.rolldownVersion) {
+				new_config.experimental = {
+					// @ts-expect-error `enableNativePlugin` only exists in `rolldown-vite`
+					enableNativePlugin: true
```

Comment:
are you sure kit wants to enable this by default? i think this should be the users choice until we are sure that it works reliably in all users projects